app/views/attachinary_files/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Listing Photos</h1>
+
+<table>
```

Comment:
Well actually i'm using the new.html.erb to display the photos I think we won't use that page. It seems its more friendly to do everything on the same page..